packages/spatial/src/camera/systems/CameraSystem.tsx

```diff
@@ -147,7 +148,9 @@ export const getMaxCamDistance = (cameraEntity: Entity, target: Vector3) => {
   raycaster.firstHitOnly = true // three-mesh-bvh setting
   raycaster.far = followCamera.maxDistance
   raycaster.set(target, targetToCamVec.normalize())
-  const hits = raycaster.intersectObjects(sceneObjects, false)
+  const hits = raycaster
+    .intersectObjects(sceneObjects, false)
+    .filter((item) => getOptionalComponent(item.object.entity, VisibleComponent) === true)
```

Comment:
this filter is unnecessary, add VisibleComponent to cameraLayerQuery

Reply:
I am blind :) thank you

Reply:
but it is cool

Reply:
or not

Reply:
now done